EntAppProject/src/main/java/com/jobook/EntAppProject/dto/UserDTO.java

```diff
@@ -0,0 +1,69 @@
+package com.jobook.EntAppProject.dto;
+
+import com.jobook.EntAppProject.model.Address;
+
+public class UserDTO {
+	private Address address;
+	private String firstName, lastName, userName, password, email, userType;
+	private Integer id, addressId;
+	public Address getAddress() {
+		return address;
+	}
+	public void setAddress(Address address) {
+		this.address = address;
+	}
+	public String getFirstName() {
+		return firstName;
+	}
+	public void setFirstName(String firstName) {
+		this.firstName = firstName;
+	}
+	public String getLastName() {
+		return lastName;
+	}
+	public void setLastName(String lastName) {
+		this.lastName = lastName;
+	}
+	public String getUserName() {
+		return userName;
+	}
+	public void setUserName(String userName) {
+		this.userName = userName;
+	}
+	public String getPassword() {
+		return password;
+	}
+	public void setPassword(String password) {
+		this.password = password;
+	}
+	public String getEmail() {
+		return email;
+	}
+	public void setEmail(String email) {
+		this.email = email;
+	}
+	public String getUserType() {
+		return userType;
+	}
+	public void setUserType(String userType) {
+		this.userType = userType;
+	}
+	public Integer getId() {
+		return id;
+	}
+	public void setId(Integer id) {
+		this.id = id;
+	}
+	public Integer getAddressId() {
+		return addressId;
+	}
+	public void setAddressId(Integer addressId) {
+		this.addressId = addressId;
+	}
+	
+	@Override
+	public String toString() {
+		return userName;
+	}
+
+}

```

Comment:
Created a new DTO that was similar to the one created in class that did not involve the super object.